package.json

```diff
@@ -197,6 +197,7 @@
     "hpagent": "^1.2.0",
     "minimist": "^1.2.8",
     "moment": "^2.30.1",
+    "node-fetch": "^3.3.2",
```

Comment:
issue: fetch is now part of nodejs it should not be added with a 3rd party library

Reply:
I added it as I got an error when commit'ing the changes. I'll check again

Reply:
I finally removed the use of the `AbortError` type, to use the same check as in the library:

https://github.com/kubernetes-client/javascript/blob/release-1.x/src/health.ts#L52